### PR TITLE
docs: group your action through namespace

### PR DESCRIPTION
**Pull request by sa-bangash**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Comment by splincode** [comment]
> This branch has conflicts that must be resolved.

File: docs/concepts/actions.md [hunk 1]
```diff
@@ -77,3 +77,26 @@ Examples:
 
 A great video on the topic is [Good Action Hygiene by Mike Ryan](https://www.youtube.com/watch?v=JmnsEvoy-gY)
 It's for NgRx, but the same naming conventions apply to NGXS.
+
+## Group your actions
+Group similar action by using namespace.
```
Comment (eranshmil) [inline docs/concepts/actions.md]: Could you add some short explanation of what are the benefits?

File: docs/concepts/actions.md [hunk 2]
```diff
@@ -77,3 +77,26 @@ Examples:
...
+
+```TS
+export namespace TodoAction {
+  export class Add {
```
Comment (eranshmil) [inline docs/concepts/actions.md]: Please add an empty line between each class and use upper case for the first character in each word of the type.

**Comment by splincode** [comment]
@eranshmil ping

**Pull request merged**